app/views/catalog/ds.html.erb

```diff
@@ -21,14 +21,9 @@
         # If the user is a repo admin or is permitted to manage both content and
         # rights, display raw xml editing
     %>
-      <%= if can?(:manage_item, @obj) &&
-             (
-               @obj.datastreams[params[:dsid]].respond_to?(:ng_xml) ||
-               params[:dsid] == 'RELS-EXT'
-             ) &&
-             params[:dsid] != 'workflows'
-            link_to "Edit #{params[:dsid]}", ds_solr_document_path(params[:id], params[:dsid], edit: 'true'), title: params[:dsid], data: { blacklight_modal: 'preserve' }
-        end %>
+      <%= if can?(:manage_item, @obj) && Settings.editable_datastreams.include?(params[:dsid]) %>
```

Comment:
for `if` blocks there should not be an `=` prefix for the erb as this doesn't return a value.

Reply:
I really shouldn't code while having trouble sleeping... fixed.